virttest/qemu_vm.py

```diff
@@ -2206,6 +2206,11 @@ def sort_key(dev):
         if bios_path:
             devices.insert(StrDev('bios', cmdline="-bios %s" % bios_path))
 
+        fix_qemu210 = params.get("fix_qemu210")
+        if fix_qemu210:
+            devices.insert(qdevices.QGlobal("i440FX-pcihost", "x-pci-hole64-fix", "off"))
+            devices.insert(qdevices.QGlobal("q35-pcihost", "x-pci-hole64-fix", "off"))
```

Comment:
I think the better place to handle this stuff is in `qemu_devices/qcontainer.py`, there is the method `machine_by_params()` and you can make it only works for `pc` and `q35` machines.

And also, I'd suggest the param name to be `vm_pci_hole64_fix`.

Thanks.

Reply:
Alright, hopefully I picked a good place in the next push.